app/containers/MessageActions/index.tsx

```diff
@@ -237,6 +239,23 @@ const MessageActions = React.memo(
 				replyInit(message, false);
 			};
 
+			const handleReplyInDM = async (message: TAnyMessageModel) => {
+				if (message.u.username) {
+					const result = await Services.createDirectMessage(message.u.username);
+					if (result.success) {
```

Comment:
Did we test what happens if the DM already exists?

Reply:
Yeap! Works properly, the user is redirected to the correct DM.